### ISSUE TITLE
Option to continue executing a macro that calls PlayMacro()

### ISSUE THREAD

**Issue by Timorio** [Reetus/ClassicAssist#156]
This would essentially treat the macro played with PlayMacro() as a function, allowing the original script to continue playing once the called macro is finished.  An example of a macro that would double as a useful function is a macro to recall home, after which the player may wish to have differing sets of actions occur.

While it's possible to write the macro to be called as an actual function inside a .py file, it'd be nice to have the flexibility to do something similar within the assistant itself.

**Comment by PoodyCZ** [comment]
> While it's possible to write the macro to be called as an actual function inside a .py file, it'd be nice to have the flexibility to do something similar within the assistant itself.

How do you achieve this btw? Make a .py file that has the function and then import it?

**Comment by Timorio** [comment]
Yes.
